src/components/BnToggle/BnToggle.vue

```diff
@@ -89,14 +96,20 @@ const attrsWithoutClass = Object.fromEntries(Object.entries(attrs).filter(([key]
         >
         <div
           class="bn-toggle__track"
-          :class="{
-            [`bn-toggle__track--checked bn-toggle__track--checked-${props.color}`]: checked,
-            [`bn-toggle__track--focus-visible bn-toggle__track--focus-visible-${props.color}`]: isFocusedVisible,
-          }"
+          :class="[
+            {
+              [`bn-toggle__track--checked bn-toggle__track--checked-${props.color}`]: checked,
```

Comment:
This is not really a change added by your PR so maybe it could go to a trello card but right now this syntax is going to create a `bn-toggle__track--checked-undefined` class here and in other places.

Reply:
You're right, I forgot about the default value